ru.json

```diff
@@ -0,0 +1,274 @@
+{
+    "name" : "Russian",

```

Comment:
Could you change this to the name of the language in Russian, and add a `name-en` field underneath as with the name in english:

`"name-en" : "Russian"`